org.kde.okular.yaml

```diff
@@ -68,12 +68,12 @@ modules:
       - soinstall
     cleanup:
       - /bin
-      - /share/ghostscript/9.23/doc/
-      - /share/ghostscript/9.23/examples
+      - /share/ghostscript/9.24/doc/
+      - /share/ghostscript/9.24/examples
     sources:
     - type: archive
-      url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs923/ghostscript-9.23.tar.xz
-      sha256: 1fcedc27d4d6081105cdf35606cb3f809523423a6cf9e3c23cead3525d6ae8d9
+      url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs924/ghostscript-9.24.tar.gz
```

Comment:
There's a tar.xz available https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs924/ghostscript-9.24.tar.xz we should use that ideally.